src/app/bs-navbar/bs-navbar.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'bs-navbar',
+  templateUrl: './bs-navbar.component.html',
+  styleUrls: ['./bs-navbar.component.scss']
+})
+export class BsNavbarComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit(): void {}
```

Comment:
remove this since it's just an empty function and remove the implements OnInit

Reply:
We end up using this function later on in Project tutorial.  This is what it looks like now:

`import { Component, OnInit } from '@angular/core';
import { AuthService } from '../auth.service';

@Component({
  selector: 'bs-navbar',
  templateUrl: './bs-navbar.component.html',
  styleUrls: ['./bs-navbar.component.scss']
})
export class BsNavbarComponent {
  constructor(public auth: AuthService) {}

  logout() {
    this.auth.logout();
  }
}
`